src/pathwayvis/services/mapoption.service.ts

```diff
@@ -291,21 +306,32 @@ export class MapOptionService {
     this.selectedCardId = id;
   }
 
-  public actionHandler(
+   public actionHandler(
```

Comment:
I guess it's a mistake. Or the indentation was off?

Reply:
mistake